### PR TITLE
docs: add quickstart docs and quick links

### PR DESCRIPTION
**Pull request by ToeFrog**
Added a quickstart section to the README file as well as quick links to jump to specific sections in the file.

### CODE REVIEW

**Review by cmjchrisjones — APPROVED** [review]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -11,6 +11,20 @@ A Visual Studio Code extension that listens to twitch chat and allows viewers to
...
 ![](./resources/screenshot-example.gif)
 
+<a name="quickstart"></a>
+## Quickstart Guide
+
+
+Install [VS Code](https://code.visualstudio.com/Download) if you don't already have it.
+
+Install [Twitch Themer](https://marketplace.visualstudio.com/items?itemName=MichaelJolley.vscode-twitch-themer) from the Visual Studio Marketplace
+Connect Twitch Themer to Twitch from the [Command Palette](#command-palette) (Ctrl + Shift + P)
```
Comment (MichaelJolley) [inline README.md:21]: Need to add an additional return here or it will render as one sentence.

File: README.md [hunk 2]
```diff
@@ -11,6 +11,20 @@ A Visual Studio Code extension that listens to twitch chat and allows viewers to
...
+
+This opens the Twitch Authentication page. Login to your Twitch account
+
+Done!
+
+<a name="twitch-chat-commands"></a>
```
Comment (MichaelJolley) [inline README.md:27]: I think we can remove all these added anchors. By default, GitHub turns the <h> tags into anchors.

**Review by MichaelJolley — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ToeFrog) [reply README.md]: Added

File: README.md (hunk 2 above)
Comment (ToeFrog) [reply README.md]: Done

**Review by MichaelJolley — APPROVED** [review]
bitmoji

**Comment by MichaelJolley** [comment]
@all-contributors please add @ToeFrog for documentation

**Pull request merged**